avm-transpiler/src/transpile.rs

```diff
@@ -0,0 +1,289 @@
+use acvm::acir::brillig::Opcode as BrilligOpcode;
+use acvm::acir::circuit::brillig::Brillig;
+
+use acvm::brillig_vm::brillig::{BinaryFieldOp, BinaryIntOp};
+
+use crate::instructions::{
+    AvmInstruction, AvmOperand, AvmTypeTag, FIRST_OPERAND_INDIRECT, ZEROTH_OPERAND_INDIRECT,
+};
+use crate::opcodes::AvmOpcode;
+use crate::utils::{dbg_print_avm_program, dbg_print_brillig_program};
+
+/// Transpile a Brillig program to AVM bytecode
+pub fn brillig_to_avm(brillig: &Brillig) -> Vec<u8> {
+    dbg_print_brillig_program(&brillig);
```

Comment:
do we want to keep these?

Reply:
They're very helpful so I think yes

Reply:
(for debugging the transpiler.... Which I expect we'll do a lot of each time Brillig changes)